.projector/workItemTemplates/engineering-fundamentals.yml

```diff
@@ -496,8 +496,8 @@ items:
         acceptanceCriteria: []
         children:
           - name:
-              As a dev crew, we have provided reproducible, actionable, specific and
-              goal-oriented CSE feedback on any blockers or potential
+              As an ISE team, we have provided reproducible, actionable, specific and
+              goal-oriented ISE feedback on any blockers or potential
```

Comment:
There is no such thing as ISE feedback... CSE feedback is a specific type of feedback that I believe is no longer used... we should probably remove this all together. 

I am wondering where this is linked as this is a bit outside of the norm for the playbook, this .projector template -- perhaps we can remove it all together, but we have to check links to it first